packages/graphql/src/interfaces/NodeInterface.ts

```diff
@@ -15,21 +11,7 @@ export function registerType(type: GraphQLObjectType) {
   return type;
 }
 
-type Loader = {
-  load: (context: GraphQLContext, id: string) => Promise<any>;
-  getLoader: () => DataLoader<string, any>;
-};
-
-export type Loaders = {
-  [key: string]: Loader;
-};
-
 export const {nodeField, nodeInterface: NodeInterface} = nodeDefinitions(
-  (globalId, context: GraphQLContext) => {
-    const {type, id} = fromGlobalId(globalId);
-    const loader: Loader = ((loaders as unknown) as Loaders)[`${type}Loader`];
-
-    return (loader && loader.load(context, id)) || null;
-  },
```

Comment:
Why are you removing this loading function?

It would make Relay unable to fetch a Node object.

Please provide a way for Relay to map from an object to the GraphQL type associated with that object.

Reply:
Sure.

[DDD_Reference_2015-03.pdf](https://github.com/animavita/animavita/files/6954903/DDD_Reference_2015-03.pdf)
[DomainDrivenDesignQuicklyOnline (1).pdf](https://github.com/animavita/animavita/files/6954904/DomainDrivenDesignQuicklyOnline.1.pdf)